packages/@cdktf/provider-generator/lib/get/generator/module-generator.ts

```diff
@@ -195,6 +198,16 @@ function parseComplexType(type: string): string | undefined {
     return `${parseType(innerType)}[]`;
   }
 
+  if (kind === "tuple") {
+    const innerTuple: string[] = innerType.slice(1, -1).split(",");
+    let tupleTypes: string[] = [];
+    innerTuple.forEach((el) => {
+      tupleTypes.push(parseType(el.trim()));
+    });
+
+    return `tuple[${tupleTypes}]`;
```

Comment:
This needs to be a valid Typescript type and more specifically a type supported by JSII.
Tuples aren't currently supported in JSII. The simple option would be to type as `any[]`. A more involved option would be to generate code for the specific types of the tuple.

Reply:
@jsteinich thanks for suggestion.
Than, for type `list(tuple([string, string, number]))` defined in tf config, should be changed as
```
any[string,string,number][]
```

If you can guide me the output form which should be returned from this function, it would be pretty helpful 🙏

Reply:
That example would probably need to be `any[][]` since tuples aren't fully supported.
The output needs to be a JSII compatible Typescript type (see https://aws.github.io/jsii/user-guides/lib-author/typescript-restrictions/ for more info).